datasets/mypos/mypos.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Introduction to the myPOS Corpus (Myanmar Part-of-Speech Corpus) for Myanmar language NLP Research and Developments"""
+
+import logging
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{khin-war-war-htike-2017-cicling,
+    title = "Comparison of Six POS Tagging Methods on 10K Sentences Myanmar Language (Burmese) POS Tagged Corpus",
+    author = "Khin War War Htike, Ye Kyaw Thu, Zuping Zhang, Win Pa Pa, Yoshinori Sagisaka and Naoto Iwahashi",
+    booktitle = "Proceedings of the 18th International Conference on Computational Linguistics and Intelligent Text Processing",
+    year = "2017",
+}
+"""
+
+_DESCRIPTION = """\
+myPOS Corpus (Myanmar Part-of-Speech Corpus) for Myanmar language NLP Research and Developments
+"""
+
+_URL = "https://raw.githubusercontent.com/hungluumfc/burmese-data/main/myPOS/"
+_TRAINING_FILE = "train.txt"
+_DEV_FILE = "dev.txt"
+_TEST_FILE = "test.txt"
+
+
+class myPOS2017Config(datasets.BuilderConfig):
+    """BuilderConfig for myPOS2017"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for myPOS2017.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(myPOS2017Config, self).__init__(**kwargs)
+
+
+class myPOS2017(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Mypos(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `mypos`

you'll have to regenerate the dataset_infos.json file after this change